internal/proto/controller/api/resources/credentialstores/v1/credential_store.proto

```diff
@@ -7,81 +7,92 @@ option go_package = "github.com/hashicorp/boundary/sdk/pbs/controller/api/resour
+  google.protobuf.StringValue ca_cert = 30 [json_name = "ca_cert", (custom_options.v1.generate_sdk_option) = true, (custom_options.v1.mask_mapping) = { this: "attributes.ca_cert" that: "CaCert" }]; // @gotags: `class:"public"`
 
   // The value to use as the SNI host when connecting to vault via TLS.
-  google.protobuf.StringValue tls_server_name = 40 [json_name = "tls_server_name", (custom_options.v1.generate_sdk_option) = true, (custom_options.v1.mask_mapping) = { this: "attributes.tls_server_name" that: "TlsServerName" }];
+  google.protobuf.StringValue tls_server_name = 40 [json_name = "tls_server_name", (custom_options.v1.generate_sdk_option) = true, (custom_options.v1.mask_mapping) = { this: "attributes.tls_server_name" that: "TlsServerName" }]; // @gotags: `class:"public"`
 
   // When set to true verification of the TLS certificate is disabled.
-  google.protobuf.BoolValue tls_skip_verify = 50 [json_name = "tls_skip_verify", (custom_options.v1.generate_sdk_option) = true, (custom_options.v1.mask_mapping) = { this: "attributes.tls_skip_verify" that: "TlsSkipVerify" }];
+  google.protobuf.BoolValue tls_skip_verify = 50 [json_name = "tls_skip_verify", (custom_options.v1.generate_sdk_option) = true, (custom_options.v1.mask_mapping) = { this: "attributes.tls_skip_verify" that: "TlsSkipVerify" }]; // @gotags: `class:"public"`
 
   // Input only. The current vault token used by this credential store for creating new credentials.
-  google.protobuf.StringValue token = 60 [json_name = "token", (custom_options.v1.generate_sdk_option) = true, (custom_options.v1.mask_mapping) = { this: "attributes.token" that: "Token" }];
+  google.protobuf.StringValue token = 60 [json_name = "token", (custom_options.v1.generate_sdk_option) = true, (custom_options.v1.mask_mapping) = { this: "attributes.token" that: "Token" }]; // @gotags: `class:"secret"`
 
   // Output only. The hmac value of the vault token used by this credential store.
-  string token_hmac = 70 [json_name = "token_hmac"];
+  string token_hmac = 70 [json_name = "token_hmac"]; // @gotags: `class:"public"`
 
   // Input only. A PEM encoded client certificate for vault with an
   // optional private key included in the bundle.  It is an error to include
   // the private key in this bundle as well as setting the certificate_key
   // field.
-  google.protobuf.StringValue client_certificate = 80 [json_name = "client_certificate", (custom_options.v1.generate_sdk_option) = true, (custom_options.v1.mask_mapping) = { this: "attributes.client_certificate" that: "Certificate" }];
+  google.protobuf.StringValue client_certificate = 80 [json_name = "client_certificate", (custom_options.v1.generate_sdk_option) = true, (custom_options.v1.mask_mapping) = { this: "attributes.client_certificate" that: "Certificate" }]; // @gotags: `class:"secret"`
```

Comment:
I think this can be public, or maybe sensitive. Nothing in a cert should be secret, and it's possible that access to the cert to know which cert was used for access could be useful to an auditor.

Reply:
Based on the comment for the field, it looks like it could contain a private key in a bundle.

Reply:
Good catch, yes. I forgot it wouldn't only be in the certificate key field.